src/data_access/FileBreachDataAccessObject.java

```diff
@@ -0,0 +1,163 @@
+package data_access;
+
+import com.mysql.cj.util.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import use_case.CheckBreach.CheckBreachDataAccessInterface;
+import use_case.LogOut.LogOutDataAccessInterface;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+public class FileBreachDataAccessObject implements CheckBreachDataAccessInterface, LogOutDataAccessInterface {
+
+    private final String API_KEY;
+    private int currentUserID;
+    private String encryptionKey;
+
+    /**
+     * Constructor for the Data Access Object of the Check Breach use case
+     * @param api_key the API key for the HaveIBeenPWNED API
+     */
+    public FileBreachDataAccessObject(String api_key) {
+        this.API_KEY = api_key;
+    }
+
+    /**
+     * Method that checks an email for breaches using HaveIBeenPWNED's API
+     *
+     * @param email the email to be checked for
+     * @return the breaches that the email has been in
+     */
+    @Override
+    public String checkEmail(String email) throws IOException, InterruptedException, URISyntaxException {
+        return getEmailResponse(email);
+    }
+
```

Comment:
Similar idea to Scan Item. Good use of resources to make this use case go smoother.